optimizer/src/main/java/com/netease/arctic/optimizer/operator/executor/BaseExecutor.java

```diff
@@ -50,4 +75,15 @@ protected long getMaxTransactionId(List<DataFile> dataFiles) {
 
     return 0;
   }
+
+  protected void executeTimeout(Closeable writer) throws Exception {
```

Comment:
What about renaming this method to `checkIfTimeout`?

Reply:
ok